### PR TITLE
chore: remove Typography

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

We're migrating off antd

## Changes

Remove all instances of Typography

## How did you test this code?

Manually for all remaining cases

### CODE REVIEW

File: frontend/src/lib/components/EntityFilterInfo.tsx [hunk 1]
```diff
@@ -11,11 +10,21 @@ interface EntityFilterInfoProps {
...
+    title: string | undefined
+    ellipsis?: boolean
+    className?: string
+    children: React.ReactNode
+}): JSX.Element {
     return (
-        <Typography.Text style={{ maxWidth: 400 }} {...props}>
-            {props.children}
-        </Typography.Text>
+        <div className={clsx('EntityFilterInfo', ellipsis && 'truncate', className)} title={title}>
```
Comment (Twixes) [inline frontend/src/lib/components/EntityFilterInfo.tsx:25]: Usually the top-level element's class in our usage of BEM is the component name, so `TextWrapper` here.
Though TBH I think we can just completely do away with `TextWrapper`, it's styling looks like unnecessary complexity, especially if `showSingleName` is false

Comment (mariusandra) [reply frontend/src/lib/components/EntityFilterInfo.tsx:25]: Seconded. We can likely get rid of `TextWrapper` completely, or then at least move the styles from the separate scss file into the component as tailwind classes.

File: frontend/src/lib/components/EntityFilterInfo.tsx [hunk 2]
```diff
@@ -11,22 +10,18 @@ interface EntityFilterInfoProps {
...
 export function EntityFilterInfo({
     filter,
     allowWrap = false,
     showSingleName = false,
     style,
 }: EntityFilterInfoProps): JSX.Element {
     if (isAllEventsEntityFilter(filter) && !filter?.custom_name) {
-        return <TextWrapper title="All events">All events</TextWrapper>
+        return (
+            <div className="text-primary font-medium whitespace-nowrap max-w-100" title="All events">
```
Comment (Twixes) [inline frontend/src/lib/components/EntityFilterInfo.tsx:21]: Looking at visual snapshot changes, this shouldn't have `text-primary font-medium`. The original `TextWrapper` didn't include those classes

**Review by Twixes — APPROVED** [review]
Nice

**Pull request merged**